soyuz_app/library/hubspot.py

```diff
@@ -63,3 +63,22 @@ def update_hubspot(self, user_hubspot_id, batch_number):
         except ApiException as e:
             capture_exception(e)
             raise ValueError("error updating hubspot")
+
+    # update funnel status on dropout from batch
+
+    def dropout_funnel_status(self, user_hubspot_id, funnel_status):
```

Comment:
it seems like `dropout_funnel_status` is just like `update_hubspot`

I think we could refactor both to have one base function that calls the API and two wrapper functions, something like:

- `update_funnel_dropout`
- `update_funnel_basics_apply`